pep-0001.txt

```diff
@@ -469,10 +489,10 @@ following RFC 2822 continuation line conventions.  Note that personal
 email addresses in PEPs will be obscured as a defense against spam
 harvesters.
 
-The BDFL-Delegate field is used to record cases where the final decision to
-approve or reject a PEP rests with someone other than the BDFL. (The
-delegate's email address is currently omitted due to a limitation in the
-email address masking for reStructuredText PEPs)
+The BDFL-Delegate field is used to record the core developer appointed by the
```

Comment:
Do we want to limit BDFL-delegate position to core devs only?  If so, do we say that explicitly?  To me, this implies that but doesn't make it explicit.  What was your intention?

Reply:
core developer -> person

Given that there are PEPs that deal with workflow, etc., it seems reasonable not to overly limit the position to core dev.

Reply:
We also use the PEP process for non-CPython activities (e.g. PyPA). Hmm, I should make that clearer in the early parts of the PEP - at the moment it's buried in an aside down in the body of the PEP about the `Discussions-To` header...

Reply:
'person' didn't quite scan right for me, so I went with 'individual'.